main.go

```diff
@@ -422,6 +422,13 @@ func main() {
 
 	RegisterHTTPReader(k)
 	go func() {
+		url := "http://"
+		if strings.HasPrefix(*listen, ":") {
+			url += "localhost"
+		}
+		url += *listen + "/"
+
+		log.Println("Starting Webserver on ", url)
```

Comment:
aaarghx… as you may have noticed from my thousands of deleted comments: I find this pretty hard to understand :) I would prefer either a `Printf("Starting webserver on http://%s/", *listen)`, even if it may be inaccurate for some values of `*listen`, or a `Println("Starting webserver on %s", *listen)`, even if the link isn't clickable.